### PR TITLE
Fixing service monitoring creation issue

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: Gowtham Shanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

File: pkg/operator/monitor/status.go [hunk 1]
```diff
@@ -48,6 +49,13 @@ func (c *ClusterStatusChecker) CheckClusterStatus(namespacedName *types.Namespac
 
 		case <-time.After(c.interval):
 			c.checkStatus(namespacedName)
+			if err := EnableServiceMonitor(ref); err != nil {
+				logger.Errorf("monitor failed err %s", err.Error())
```
Comment (sp98) [inline pkg/operator/monitor/status.go:53]: @GowthamShanmugam Do we need to handle this error other than just logging it?

Comment (GowthamShanmugam) [reply pkg/operator/monitor/status.go:53]: ack, will fix this

**Pull request merged**